dts/bindings/adc/st,stm32-adc.yaml

```diff
@@ -27,6 +27,14 @@ properties:
     pinctrl-names:
       required: true
 
+    st,prescaler:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Clock prescaler at the input of the adc
+        Could be in range [0 .. 0xF] ; default is 0 (not divided)
+
```

Comment:
Does it make sense to use an enum type here?

Reply:
adding `Valid range: 0 - 15`

Reply:
adding `Valid range: 0 - 15`